src/language-css/printer-postcss.js

```diff
@@ -316,9 +316,9 @@ function genericPrint(path, options, print) {
           : "",
         node.attribute.trim(),
         node.operator ? node.operator : "",
-        node.value
+        typeof node.value !== "undefined"
           ? quoteAttributeValue(
-              adjustStrings(node.value.trim(), options),
+              node.quoteMark ? adjustStrings(node.raws.value, options) : node.value,
               options
             )
           : "",
```

Comment:
This entire approach to space trimming is hostile to intervening comments as it does not consider the `raws` setting for spaces which is where the comments are stored.

Reply:
I don't have the context on this, but there must be a reason for doing this. I didn't touch this part of the code that much but in my (little) experience, the AST output on CSS in general is very brittle. We have all sorts of hacks to prevent printing anything wrong.

Reply:
I added a commit on this other branch with some ideas for how this code could work better. But I don't see it working nicely with the tests, so I don't really understand what's the issue there. Hopefully it will add some thoughts for how you can do this differently without dropping intervening comments within the attribute.

https://github.com/chriseppstein/prettier/commits/selector-parser-v5-comment-preservation